carmin.yaml

```diff
@@ -171,8 +171,7 @@ paths:
           schema:
             type: array
             items:
-              type: string
-              format: url
+              $ref: '#/definitions/Path'
```

Comment:
Of course we need a method that return Path, but it could be useful to still have a method that return direct URLs ?

Reply:
URLs can be directly extracted from the Path object. I don't think we should manipulate URLs without Paths. Why would it be useful?

Reply:
That's true, I thought that needed one additional request but I was wrong.
So, to resume, getExecutionResults returns :
`[  { "uri" : "vip://vip.fr/path/to/file", [...] } ]`
And then the carmin client can download with 
`https://carmin-server.com/path/content?uri=vip://vip.fr/path/to/file`